shared/src/main/java/com/vaadin/shared/ui/Connect.java

```diff
@@ -98,6 +98,12 @@
         /**
          * Loaded to the client only if needed.
          */
-        LAZY
+        LAZY,
+        /**
+         * Completely left out of the widgetset.
+         *
+         * @since 8.1
```

Comment:
The commit "Add LoadStyle.NONE for completely omitting a connector" is a minor new feature and breaks compatibility and already has @since 8.1, so I'd drop it from the branch.